simple-oidc-provider/FULL_IMAGE_NAME

```diff
@@ -1 +1 @@
-adharatech/simple-oidc-provider:0.1.3
+adharatech/simple-oidc-provider:0.1.4
```

Comment:
`0.1.4-rc1` ?

Reply:
I don't want to generate more patch versions that are actually broken :(

Reply:
fixed!